CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Changed
+- Update wet day frequency correction again to raise the "low" value used to the threshold value / 10 (PR #159, @dgergel)
```

Comment:
I'd have a look at the recent version of CHANGELOG.md.

It looks like this addition is being added to a section that has now been released under v0.14.0.